Resources/Prototypes/Entities/Clothing/Head/misc.yml

```diff
@@ -171,3 +171,15 @@
   - type: Clothing
     sprite: Clothing/Head/Misc/squiddy.rsi
   - type: IngestionBlocker
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHatDonJuan
+  name: Don Juan
```

Comment:
I named it that way because richard for example (the chicken head) is named Richard and there was no indication of a mask

Reply:
yeah but this will be a mask in-game and not some representation of a messed up thing in the head of the protagonist so it needs to follow the same model as the rest of the mask